tasmota/xsns_69_opentherm_protocol.ino

```diff
@@ -32,6 +32,7 @@ typedef union {
         uint8_t notSupported : 1; // If set, boiler does not support this command
```

Comment:
Since `notSupported` is used to skip one-off commands as well, I think it worth renaming it into something like
```
uint8_t skip : 1; // If set, the command won't be transmitted to the boiler.
```

Reply:
may be its not a bad idea